pkg/handler-launcher-com/cmd/v1/cmd.proto

```diff
@@ -22,6 +22,7 @@ message VMI {
 
 message VMIRequest {
   VMI vmi = 1;
+  bytes options = 2;
```

Comment:
I see you just did the same as for MigrationRequest, but I am wondering why we use `bytes` here?
Wouldn't it be easier to introduce a `SyncOptions` message, with a `SMBIOS` message, which contains the SMBIOS fields? I think that makes maintenance easier, as gRPC handles (de)-serialization, and new and deleted messages and fields. At the end that was the reason to introduce gRPC for handler/launcher communication.
@vladikr do you remember why we use bytes for the migration options?

Reply:
I also feel having a separate `VMIOptions` struct is better here, I was just trying to follow Migrations so `bytes` was followed without any reason.

Reply:
Actually, I think now I get why `byte` was used
if `options` [here](https://github.com/kubevirt/kubevirt/pull/2472/files#diff-db6785c5e48bfb0b8649e70a67e2d047R257) is not `bytes`, `optionsJson` would not be able to assign it directly
We'll either need a dumb wrapper struct on the proto side, or will need to extract fields of options like `SMBios` and have a separate `json` for it (which I think we should not do, to keep the options generic here)
So `bytes` is more easy way to pass the whole `optionsJson` from the client

Reply:
Mh, tbh I do not see why we need bytes and json. We could just use the struct generated for gRPC (will be in in cmd.pb.go as soon as you declare the messages and run make generate) in vm.go instead of the struct you are defining yourself atm in types.go.